### PR TITLE
Start with the first asset selected

### PR DESCRIPTION
**Pull request by manuq**
https://phabricator.endlessm.com/T21625

### CODE REVIEW

File: main_renderer.js [hunk 1]
```diff
@@ -75,6 +75,14 @@ $(document).ready(() => {
         }
     });
 
+    setTimeout(function() {
```
Comment (ptomato) [inline main_renderer.js]: Why the timeout?

Comment (manuq) [reply main_renderer.js]: It doesn't work without it. Even a timeout of 1 millisecond makes it work. I will investigate why.

Comment (manuq) [reply main_renderer.js]: @ptomato this is needed lo pause the Javascript execution and let the browser render. If the browser is not rendered, these selectors like `'#documentList tr'`return nothing. A setTimeout(fn, 0) worked as well, and setImmediate() too. I switched to setImmediate() but I don't really know which one is better.

Comment (sgnn7) [reply main_renderer.js]: I think `window.onload` or jQuery [`ready`](https://learn.jquery.com/using-jquery-core/document-ready/) is what you need to use here.

Comment (manuq) [reply main_renderer.js]: @sgnn7 actually this code is already inside a `$(document).ready()` callback.

Comment (sgnn7) [reply main_renderer.js]: Hmm... good point ...but ....then how are the selectors for `on('click')` working right above it?

Comment (ptomato) [reply main_renderer.js]: Yeah, this seems weird. It would be best to dig a little bit more into why this is happening.

**Pull request merged**